packages/backend/src/webmonetization/model.ts

```diff
@@ -0,0 +1,27 @@
+import { BaseModel } from '../shared/baseModel'
+
+export class WebMonetization extends BaseModel {
+  public static get modelPaths(): string[] {
+    return [__dirname]
+  }
+
+  public static get tableName(): string {
+    return 'webMonetization'
+  }
+
+  static get idColumn(): string {
+    return 'accountId'
+  }
+
+  public accountId!: string
+  public currentInvoiceId!: string
+
+  public $beforeInsert(): void {
+    this.createdAt = new Date().toISOString()
+    this.updatedAt = new Date().toISOString()
+  }
+
+  public $beforeUpdate(): void {
```

Comment:
This is the default implementation on the parent class BaseModel already.

Reply:
This was to get around the stuff above as initially I wasn't extended the baseModel. Will cleanup